server/controllers/page_tags.js

```diff
@@ -118,10 +119,29 @@ const destroy = (req,res) => {
     })
 }
 
+const retrieveArticles = (req,res) => {
+    const id = req.params.pageTagId;  
+    console.log(`Id = ${id}`);
+    PageTags
+    .findByPk(id,{
```

Comment:
use where statement here. Same goes for your earlier pull request. 
(I made the mistake in my associations work. Sorry, for that. I will fix it. )
![image](https://user-images.githubusercontent.com/26373555/65376371-63ebc600-dcb8-11e9-9feb-4a928f682e84.png)